<launch>
  <arg name="camera_topic" default="/uav1/mobius_front/image_raw"/>
  <arg name="bag_file" default="$(env PWD)/forestnav.bag"/>
  <arg name="image_location" default="$(env PWD)/imagestream/frame%06i.png"/>
  <arg name="image_type" default="compressed"/>

  <node pkg="rosbag" type="play" name="rosbag" args="$(arg bag_file)"/>
  <node name="extract" pkg="image_view" type="extract_images" respawn="false" output="screen">
    <remap from="image" to="$(arg camera_topic)"/>
    <param name="filename_format" value="$(arg image_location)"/>
    <param name="image_transport" value="$(arg image_type)"/>
  </node>
</launch>